apache/trafficcontrol
# Update CDN in a Box to CentOS 8

## What does this PR (Pull Request) do?


~Depends on #5262 (merged), drafted until #5262 is closed.~


This PR bases the CDN in a Box Docker images off of CentOS 8 by default but allows users to continue to use CentOS 7 if they want by specifying the `CENTOS_VERSION` `docker-compose` build arg.

## Which Traffic Control components are affected by this PR?


- CDN in a Box
- Documentation

## What is the best way to verify this PR?

* Build and run CDN in a Box normally, make sure it works.
  ```shell script
  export DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1
  docker-compose build --parallel
  docker-compose down -v
  docker-compose up
  ```

* Build and run CDN in a Box with CentOS 7 images, make sure it works.
  ```shell script
  export DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1
  docker-compose build --parallel --build-arg CENTOS_VERSION=7
  docker-compose down -v
  docker-compose up
  ```

## The following criteria are ALL met by this PR


- [x] This PR changes CDN-in-a-Box, which is a test
